teachertool/src/components/ShareLinkInput.tsx

```diff
@@ -59,6 +68,7 @@ export const ShareLinkInput: React.FC<IProps> = () => {
                 preserveValueOnBlur={true}
                 autoComplete={false}
                 handleInputRef={setInputRef}
+                initialValue={text}
```

Comment:
If we have this now, since we have text set to the empty string, will that mean that if users get to the teacher tool via something other than clicking the evaluate button, they'll see an empty text box in the project area or does the placeholder show?